pandas/_typing.py

```diff
@@ -91,6 +91,12 @@
 PandasScalar = Union["Period", "Timestamp", "Timedelta", "Interval"]
 Scalar = Union[PythonScalar, PandasScalar]
 
+# numpy compatible types
+NumpyArrayLike = Union[np.ndarray, Sequence[PythonScalar]]
```

Comment:
why PythonScalar instead of e.g. Scalar?

Reply:
Because `NumpyArrayLike` is what `numpy` will accept and preserve type, and the `PandasScalar` types won't accept that.  Although now I see that that was a temporary thing I put in there, so I will delete it in the next commit.